File: FrameIncam.Domains/Models/Master/Geo/MasterGeo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace FrameIncam.Domains.Models.Master.Geo
{
    [Table("master_geo")]
    public class MasterGeo : Model
    {
        [Column("Geo_Code")]
        public long? GeoCode { get; set; }
        [Column("Geo_Name")]
        public string GeoName { get; set; }
        [Column("Geo_Head")]
        public int? GeoHead { get; set; }
        [Column("Geo_Level")]
        public int? GeoLevel { get; set; }
        [Column("Geo_type")]
        public int? GeoType { get; set; }
        [Column("Geo_TinNo")]
        public int? GeoTinNo { get; set; }
        [Column("Geo_StateCode")]
        public string GeoStateCode { get; set; }
        [Column("Geo_Latitude", TypeName = "decimal(18, 10)")]
        public decimal? GeoLatitude { get; set; }
        [Column("Geo_Longitude", TypeName = "decimal(18, 10)")]
        public decimal? GeoLongitude { get; set; }
        [Column("Geo_Isactive")]
        public int? GeoIsactive { get; set; }
    }
}

File: FrameIncam.Client/ng/domains/master/geo/masterGeo.js
(function ($, ng, debugMode, undefined) {
    "use strict";

    // --------------------------------------------------------------------------------
    // Dependencies
    // --------------------------------------------------------------------------------
    var module = ng.module("app");

    // --------------------------------------------------------------------------------
    // Model
    // --------------------------------------------------------------------------------
    module.run([
        "services.utils",
        "services.models",
        function (
            p_utils,
            p_models
        ) {
            // --------------------------------------------------------------------------------
            // Namespace
            // --------------------------------------------------------------------------------
            var fincam = p_models.namespace("fincam");
            var geo = p_models.namespace("fincam.master.geo");

            // --------------------------------------------------------------------------------
            // Model
            // --------------------------------------------------------------------------------
            p_models.class({
                namespace: geo,
                name: "masterGeo",
                base: {
                    namespace: fincam,
                    name: "model"
                },
                properties: {
                    id:  "int",
                    geoCode: "string",
                    geoName:  "string",
                    geoHead: "string",
                    geoLevel: "string",
                    geoType: "string",
                    geoTinNo: {
                        name: "int",
                        nullable: true
                    },
                    geoStateCode: "string",
                    geoLatitude:
                    {
                        name: "number",
                        nullable: true
                    },
                    geoLongitude: {
                        name: "number",
                        nullable: true
                    },
                    isactive: "int"
                },
                prototype: new function () {
                    var self = this;
                    return self;
                }
            });
        }
    ]);

})(jQuery, angular, document.children[0].hasAttribute("debug"));